BaseLib/FileTools.cpp

```diff
@@ -57,16 +57,6 @@ void truncateFile( std::string const& filename)
     ofs.close();
 }
 
-std::ofstream createBinaryFile(std::string const& file_name)
-{
-    std::ofstream os(file_name, std::ios::binary);
-    if (!os)
-    {
-        OGS_FATAL("Could not open file '%s' for output.", file_name.c_str());
-    }
-    return os;
```

Comment:
What's the problem? Missing copy elision?

Reply:
The move operator is not implemented as I understood. https://stackoverflow.com/a/28775806

Reply:
Another one not working on gcc < 5:
```c++
struct A { int a; int b = 0; };
void f() { A a = {2, 3}; }
// or
A g() { return {4, 5}; }
```
because the struct with default initializers cannot be constructed from initializer list; see https://stackoverflow.com/a/37777814
Happened here: https://jenkins.opengeosys.org/blue/organizations/jenkins/ufz%2Fogs/detail/PR-2159/13/pipeline/

I think it's time to move to minimum gcc 5.3 (5.2 had an internal compiler error in the other PR.)...